monitoring/managing-metrics.adoc

```diff
@@ -14,7 +14,12 @@ include::modules/monitoring-setting-up-metrics-collection-for-user-defined-proje
 include::modules/monitoring-deploying-a-sample-service.adoc[leveloffset=+2]
 include::modules/monitoring-specifying-how-a-service-is-monitored.adoc[leveloffset=+2]
 
+.Additional resources
```

Comment:
Suggestion: Need for an abstract, or introduction, for this assembly as per the mod docs guidelines.

Reply:
Added the abstract label for the assembly.